users/views.py

```diff
@@ -1,3 +1,9 @@
-# from django.shortcuts import render
+from django.shortcuts import render
 
-# Create your views here.
+
+def profile(request):
```

Comment:
You should add `@login_required` decorator here, you can see reference to it in Corey Schafer's guide here:
https://youtu.be/3aVqWaLjqS4?list=PL-osiE80TeTtoQCKZ03TU5fNfx2UY6U4p&t=1578

Reply:
Added